pom.xml

```diff
@@ -118,6 +118,7 @@
         <module>caf-audit-service-client</module>
         <module>caf-audit-service-server-stub-source</module>
         <module>caf-audit-service</module>
+        <module>caf-audit-service-testing-audit</module>
```

Comment:
I have renamed this to "-auditlog" in the latest commit. Changing it to caf-audit-service-testing sounds like it contains tests for the audit service.